### ISSUE TITLE
Diagram ok on desktop browser, syntax error on mobile

### ISSUE THREAD

**Issue by justmars** [mermaid-js/mermaid#4415]
### Description

Using the latest rc: `mermaid@10.2.0-rc.3`, Mermaid works fine on desktop browser: both full width viewport and responsive viewports:

<details>
<summary>What appears on desktop</summary>

<img width="734" alt="Screenshot 2023-05-21 at 1 08 29 AM" src="https://github.com/mermaid-js/mermaid/assets/39538958/40b68fc0-0c38-4a97-9df6-64dc835738c2">
</details>

When viewing from a mobile device, however, e.g. iPhone/iPad, a syntax error appears instead:

<details>
<summary>What appears on mobile</summary>

<img width="734" alt="Screenshot 2023-05-21 at 1 08 29 AM" src="https://github.com/mermaid-js/mermaid/assets/39538958/1921669f-ae90-44fb-9140-c52993a950d2">
</details>

### Steps to reproduce

1. Visit [mv3.dev/](https://mv3.dev/eli5-django) via desktop device
2. Visit [mv3.dev/](https://mv3.dev/eli5-django) via mobile device
3. Compare results

### Code Sample

```text
<script type="module">
      import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10.2.0-rc.3/+esm';
      mermaid.initialize({ startOnLoad: false, theme: 'dark'});
      await mermaid.run({
        querySelector: '.language-mermaid',
      });
    </script>
```

### Setup

- Mermaid version: 10.2.0-rc.3
- Browser and Version: Edge 113, Chrome 113, Safari 16.5

**Comment by frozenfrank** [comment]
Two years later, this no longer appears to be an issue. I just opened the website on an iPad, and it renders just fine. I recommend closing this issue as resolved.

<img width="1180" alt="Image" src="https://github.com/user-attachments/assets/885826f6-f5c4-4a23-8add-47e2ace88227" />
